src/Codeception/Module/Yii2.php

```diff
@@ -294,26 +297,21 @@ protected function onReconfigure(): void
     }
 
     /**
-     * Adds the required server params.
-     * Note this is done separately from the request cycle since someone might call
-     * `Url::to` before doing a request, which would instantiate the request component with incorrect server params.
+     * @return array<string, mixed>
```

Comment:
Isn't every value a string? We can be even more specific and use array shape.

Reply:
It should be, but it could be misconfigured.

Since it is possible to write any type of value in $this->config['entryScript'] and we are not validating the type of data received anywhere I get what you see in the screenshot.

![imagen](https://github.com/user-attachments/assets/d48bb976-c74b-4621-8952-00bf754abe4e)

I changed it to use array shape, I hope that's what you meant.

the port was returned manually as string, but from `parse_url` it is returned as int, let me know if you prefer an explicit cast to string `(string)` or if you are satisfied with `string|int`.